api/controllers/HomeController.js

```diff
@@ -8,7 +8,8 @@
 module.exports = _.mapValues({
   async index (req, res) {
     const boxes = await Box.find({owner: req.user.name});
-    res.view('home/view', {boxes});
+    const prefs = await UserPreferences.find({user: req.user.name});
```

Comment:
Re. rendering this stuff serverside -- I think we shouldn't touch the serverside code at all for the purposes of creating the client, and instead use the API to get the preferences (via the `/preferences` endpoint). 

Advantages of not touching the serverside code:

* Our client and server end up less coupled.
* We already have a JSON API for this info, so we don't need to maintain two separate methods to access it.
* There are already thorough tests in place for the JSON API, particularly for security concerns (e.g. not disclosing the existence of a user's unlisted boxes). If the server is also sending database information via the HTML site, then we have twice as many things to test, and we're much more likely to mess it up somewhere and leak private info, like what happened [here](/pokemontrades/flairhq/pull/340/files#diff-c027fe067a643ce60d09a3aff3f4cde9L14) with FlairHQ. The safest way to keep the info secure is to have as little attack surface as possible.

tl;dr: We already have a RESTful API, let's use it.